ffcv/transforms/utils/fast_crop.py

```diff
@@ -1,7 +1,58 @@
 import ctypes
-from numba import njit
+from numba import njit, prange
 import numpy as np
-from ...libffcv import ctypes_resize, ctypes_rotate, ctypes_shear, ctypes_add_weighted
+from ...libffcv import ctypes_resize, ctypes_rotate, ctypes_shear, ctypes_add_weighted, ctypes_equalize
+
+
+"""
+Custom equalize -- equivalent to torchvision.transforms.functional.equalize,
+but probably slow -- scratch is a (channels, 256) uint16 array.
+"""
+@njit(parallel=True, fastmath=True, inline='always')
+def equalize(source, scratch, destination):
+    for i in prange(source.shape[-1]):
+        scratch[i] = np.bincount(source[..., i].flatten(), minlength=256)
```

Comment:
Unfortunate that `np.bincount` doesn't have an `out` argument...

Reply:
A numba version should be pretty fast and relatively easy to implement no ?

Reply:
Yeah, good idea. I'll try to add that in the near future.